Documentation/quickstart.md

```diff
@@ -6,7 +6,7 @@ weight: 2
 # Quickstart Guide
 
 Welcome to Rook! We hope you have a great experience installing the Rook storage platform to enable highly available, durable storage
-in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://Rook-io.slack.com).
+in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://Rook-io.slack.com). Sign up for slack [here](https://rook-slackin.herokuapp.com/).
```

Comment:
Could you make the `Rook.io` to `rook.io` while you are at it.
I think this `Sign up for our [Slack](https://rook-slackin.herokuapp.com).` makes it sound better.